integreat_cms/cms/views/mixins.py

```diff
@@ -132,7 +132,6 @@ def get_context_data(self, **kwargs):
             and self.request.region.summ_ai_enabled
             and kwargs.get("language_slug")
             in [
```

Comment:
```suggestion
            == settings.SUMM_AI_EASY_GERMAN_LANGUAGE_SLUG
```
Maybe we don't need `in` for a single value?